example_client/multi_inputs.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env python
+#
+# Copyright 2019, Intel Corporation
+#
+# Multi-threaded sample to run a RMNet & SSDMobilenet v2 that will
+# detect only person, bike and vehicle (change the output parsing
+# for more classes)
+#
+# For this example to work, you will need to follow the steps to run
+# hddldaemon & OVMS setup seperately. Below usage instructions assume you
+# have got this setup to work with the right network with the HDDL
+#
+# Example usage:
+# RMNet: python3.6 multi_inputs.py -n "RMNet" -l "data" -o "detection_out"
+# -d 1024 -i 127.0.0.1 -p 9001 -c 1
+# -f /var/repos/github/sample-videos/person-bicycle-car-detection.mp4
+# SSDMobileNet: python3.6 multi_inputs.py -n "SSDMobileNet" -l "image_tensor"
+# -o "DetectionOutput" -d 300 -i 127.0.0.1 -p 9001 -c 1
+# -f /var/repos/github/sample-videos/person-bicycle-car-detection.mp4
+
+from __future__ import print_function
+from argparse import ArgumentParser, SUPPRESS
+from tensorflow_serving.apis import predict_pb2
+from tensorflow_serving.apis import prediction_service_pb2_grpc
+from time import time, sleep
+
+import sys
+import os
+import cv2
+import grpc
+import threading
+import numpy as np
+import logging as log
+import tensorflow.contrib.util as tf_contrib_util
+
+# global data (shared between threads & main)
+CLASSES = ["None", "Pedestrian", "Vehicle", "Bike", "Other"]
+COLORS = [(255, 255, 255), (255, 0, 0), (0, 255, 0), (0, 0, 255), (128, 128, 128)]
+SRC_TYPE = ["Camera", "Video"]
+
+exit_ok = False         # manage thread loop
+cam_width = 640         # camera width
+cam_height = 480        # camera height
+cam_fps = 30            # camera speed
+
+#####################################################################################
+
+def build_argparser():
+  parser = ArgumentParser(add_help=False)
+  args = parser.add_argument_group('Options')
+  args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+        help='Show this help message and exit.')
+  args.add_argument('-n', '--network_name', required=True,
+        type=str, help='Network name')
+  args.add_argument('-l', '--input_layer', required=True,
+        type=str, help='Input layer name')
+  args.add_argument('-o', '--output_layer', required=True,
+        type=str, help='Output layer name')
+  args.add_argument('-d', '--input_dimension', required=True,
```

Comment:
Not sure about this naming. I was thinking about something like `--frame-size` with description in help like 
`"Input frame width and height that matches used model"`. 
I think "dimension" might be confused with numpy arrays shapes. What do you think?

Reply:
Done